<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInfoDialog</class>
 <widget class="QDialog" name="EditInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bottomHorizontalLayout" stretch="2,1,2">
    <item>
     <widget class="QPushButton" name="confirmPushButton">
      <property name="text">
       <string>确 定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="text">
       <string>取 消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>31</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftVerticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pre_pswLabel">
        <property name="text">
         <string>旧密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="new_pswLabel">
        <property name="text">
         <string>新密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="psw_confirmLabel">
        <property name="text">
         <string>确认新密码：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightVerticalLayout">
      <property name="spacing">
       <number>25</number>
      </property>
      <item>
       <widget class="QLineEdit" name="pre_pswLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="new_pswLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="psw_confirmLneEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
